### PR TITLE
feat: add tailwind-audio theme

### LINKED ISSUE #23: Add Tailwind CSS audio theme
https://media-chrome.mux.dev/examples/vanilla/themes/tailwind-audio-theme/
https://github.com/muxinc/media-chrome/blob/main/examples/vanilla/themes/tailwind-audio-theme/index.html

### PR DESCRIPTION
**Pull request by luwes**
fix #23

### CODE REVIEW

File: themes/tailwind-audio/template.html [hunk 1]
```diff
@@ -0,0 +1,227 @@
+<style>
+  <%- include('/styles.css') -%>
```
Comment (luwes) [inline themes/tailwind-audio/template.html:2]: ejs include here

File: site/app/_components/ThemePreview.tsx [hunk 2]
```diff
@@ -19,14 +20,15 @@ export default function ThemePreview(props: ThemePreviewProps) {
   return (
     <>
       <div className="border-ctx border -m-0.5px relative grid gap-x-2 gap-y-1 p-1 pb-2 md:px-2 md:py-1.5">
-        <div className="relative bg-white">
+        <div className="relative">
```
Comment (heff) [inline site/app/_components/ThemePreview.tsx:23]: I think audio players need more vertical margin here. It almost gets lost between video players.
image

File: site/themes/tailwind-audio.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (heff) [inline site/themes/tailwind-audio.md:1]: I would be nice if these were stored next to the themes, yeah? Not a high priority.

File: themes/tailwind-audio/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "prebuild": "tailwindcss -i ./styles.css --minify -o ./dist/styles.css",
+    "build": "build-theme"
+  },
+  "peerDependencies": {
+    "media-chrome": ">=1.0.0"
+  },
+  "devDependencies": {
+    "@tailwindcss/container-queries": "^0.1.0",
+    "build-theme": "^0.0.2",
+    "tailwindcss": "^3.2.4"
```
Comment (heff) [inline themes/tailwind-audio/package.json:36]: First external dependency! (I think) Any issues come up related to that?

File: themes/tailwind-audio/template.html (hunk 1 above)
Comment (heff) [reply themes/tailwind-audio/template.html:2]: What are the implications of doing this? Like, you couldn't use this template directly with <media-template>, right?

Could we just inline the tailwind CSS in the template.html?

File: themes/tailwind-audio/template.html [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+  audio
+  defaultsubtitles="{{defaultsubtitles}}"
+  defaultduration="{{defaultduration}}"
+  gesturesdisabled="{{disabled}}"
+  hotkeys="{{hotkeys}}"
+  nohotkeys="{{nohotkeys}}"
+  defaultstreamtype="on-demand"
+  class="@container block w-full shadow-xl shadow-black/5"
+  style="
+    --media-background-color: transparent;
```
Comment (heff) [inline themes/tailwind-audio/template.html:92]: Should any of these become defaults for media chrome when using audio?

File: themes/tailwind-audio/template.html [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+      >
+        <use href="#backward" />
+      </svg>
+    </media-seek-backward-button>
+
+    <media-play-button
+      class="h-10 w-10 p-2 mx-3 rounded-full bg-slate-700 hover:bg-slate-900 focus:outline-none focus:ring-slate-700 focus:ring-2 focus:ring-offset-2"
+    >
+      <svg slot="play" aria-hidden="true" class="relative left-px">
+        <use href="#play" />
```
Comment (heff) [inline themes/tailwind-audio/template.html:139]: This is interesting. Is there a benefit to this sprite approach for SVGs vs. just having the SVG path here? I guess if you're using it twice in the theme?

**Review by heff — COMMENTED** [review]
Looking good. 👍 A few questions.

File: themes/tailwind-audio/template.html (hunk 1 above)
Comment (luwes) [reply themes/tailwind-audio/template.html:2]: it does that, it gets inlined by the build-theme CLI. 
`template.html` is the source file now and the build script creates a `dist/media-theme.html` file and a `dist/media-theme.js` file

File: themes/tailwind-audio/template.html (hunk 5 above)
Comment (luwes) [reply themes/tailwind-audio/template.html:92]: good call, probably yes.

File: site/themes/tailwind-audio.md (hunk 3 above)
Comment (luwes) [reply site/themes/tailwind-audio.md:1]: yea no, this is intentional. 

`site/themes` hold the site content.
`[root]/themes` are the player.style theme packages (NPM).

File: themes/tailwind-audio/package.json (hunk 4 above)
Comment (luwes) [reply themes/tailwind-audio/package.json:36]: yep. it's a dev dependency so not that complex yet. just some shuffling I had to do with the build scripts.

for regular dependencies we'll have to add some logic to the build script so those ESM imports are added at the top of the media-theme.js file...

File: themes/tailwind-audio/template.html (hunk 6 above)
Comment (luwes) [reply themes/tailwind-audio/template.html:139]: indeed, benefit is for re-use and maybe more readable HTML.

**Pull request merged**